frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -161,9 +166,15 @@ def _create_page(self, obj):
         db_page.fingerprint = obj.meta['fingerprint']
         db_page.state = Page.State.NOT_CRAWLED
         db_page.url = obj.url
-        db_page.depth = 0
         db_page.created_at = datetime.datetime.utcnow()
         db_page.meta = obj.meta
+
+        if not isinstance(obj, frontera_response):
```

Comment:
I think backend database is a storage of metadata records. The *record* has a wider meaning, than just requests. The important use of records is generation of requests to fetch. But, there are more uses: 
 - matching them with responses (see https://github.com/scrapinghub/frontera/issues/63#issuecomment-138244380), 
 - using them during canonical URL identification (it could require some per-host level operations),
 - additional metadata storage (for example two types of score: score for prioritization, and some content-based score, calculated during *response* parsing, used for other purposes than prioritization),
 - for statistics: what was crawled, when, what errors were encountered, etc, 
 - serve as entities to be shown in external UI.

So you can see these records as requests, but I would say this kind-of limits the whole concept.